static/src/javascripts-legacy/projects/common/modules/atoms/youtube.js

```diff
@@ -209,6 +225,15 @@ define([
         });
     }
 
+    function updateImmersiveButtonPos() {
+        var playerHeight = document.querySelector('.immersive-main-media__media .youtube-media-atom').offsetHeight;
+        var headline = document.querySelector('.immersive-main-media__headline-container');
+        var headlineHeight = (headline ? headline.offsetHeight : 0);
```

Comment:
Not a biggy but the brackets around the evaluation aren't required here, it still works with or without them.